.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build images for Docker Hub
+on:
+  schedule:
+    - cron: '0 18 * * 1' # every monday at 18:00
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Docker Login
+        run: docker login -u "${{ secrets.DOCKER_USER }}" -p "${{ secrets.DOCKER_PASSWORD }}"
+      - name: Set Tag
+        run: echo ::set-env name=DATE::$(date "+%Y%m%d")
+      - name: Build Image
+        run: docker build -t 'projectsyn/suc-ubuntu-bionic:'$DATE .
```

Comment:
Maybe we should explicitly set the `ubuntu_version=18.04` argument to be more explicit?

Reply:
append the version to the tag you say? example 20200505_18.04?

Reply:
no, just add the argument when calling `docker build --build-arg ubuntu_version=18.04`.
So that if we change the default this is still correct.

Reply:
good idea